src/components/inputs/filled-icon-input/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { TextField, InputAdornment } from "@mui/material";
+
+const FilledIconInput = ({
+  icon = null,
+  placeHolder = "place holder",
+  isIconStart = true,
+  text = null,
+  handler,
+}) => {
+  // {type} prop can be: "filled", "outlined", or "standard"
+  // {isStart} prop is a boolen that shows if the icon displays at start or end of input
+
+  return (
+    <TextField
+      fullWidth
+      variant={"filled"}
+      value={text}
+      label={placeHolder}
+      //   onChange={(e) => {
+      //     handler(e);
+      //   }}
```

Comment:
I believe you need onChange in TextField why it's commented?

Reply:
To keep everything related to UI at the moment, also I don't want to provide handlers for now.